.gitignore

```diff
@@ -9,4 +9,5 @@ static-collected
 static/.DS_Store
 
 # pycharm
-.idea/
\ No newline at end of file
+.idea/sendgrid.env
```

Comment:
Let's change this back to `.idea/` it's adding pycharm related files to revision control